samples/nrf_rpc/protocols_serialization/client/src/main.c

```diff
@@ -8,19 +8,38 @@
 
 #include <nrf_rpc.h>
 
+NRF_RPC_GROUP_DECLARE(ot_group);
 LOG_MODULE_REGISTER(nrf_ps_client, CONFIG_NRF_PS_CLIENT_LOG_LEVEL);
 
 static void err_handler(const struct nrf_rpc_err_report *report)
 {
 	LOG_ERR("nRF RPC error %d ocurred. See nRF RPC logs for more details", report->code);
 }
 
+static void bound_handler(const struct nrf_rpc_group *group)
```

Comment:
Should we do it for all groups?

Reply:
I will add it for BT RPC as well. Overall, this is just an example, as in this sample we don't autostart OT/BLE stack so we cannot do any actions to restore the state of the server except for letting a user know about the restart and let them re-type shell commands if needed. In real applications, a user will be able to use the handler to execute a custom action to bring OT/BLE stack up again.